common/src/fixpath.c

```diff
@@ -224,7 +245,16 @@ char *fix_at_file(char *in)
 
   tmpdir = getenv("TMP");
   if (tmpdir == NULL) {
+    // Set temporary directory to the cygwin TMP variable
+    struct stat buf;
     tmpdir = "c:/cygwin/tmp";
+    if ((stat("c:/cygwin", &buf) != 0) || (buf.st_mode & S_IFDIR == 0)) {
```

Comment:
Should `stat("c:/cygwin", &buf)` just be `stat(tmpdir, &buf)`?  is there a reason we're stat'ing the cygwin and not the full path?

Reply:
`stat(tmpdir, &buf)` works too. Full path would do too.